### ISSUE TITLE
minimum version

### ISSUE THREAD

**Issue by fukemy** [Yoonit-Labs/ios-yoonit-facefy#7]
hi, i got this problem after run pod:
```
[!] CocoaPods could not find compatible versions for pod "YoonitFacefy":
  In Podfile:
    YoonitFacefy

Specs satisfying the `YoonitFacefy` dependency were found, but they required a higher minimum deployment target.

[!] Automatically assigning platform `iOS` with version `13.0` on target `CallKitExample` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

[!] Automatically assigning platform `iOS` with version `13.0` on target `Sharing` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.
```
Can u provide the minimum version? Thanks

**Comment by fukemy** [comment]
hi, i found your example project is setting 14.1, i want to ask can i set it to 12.1, or minimum 13.0, because we have a lot of users that using ios version ~ 13.0

**Comment by TeruyaHaroldo** [comment]
Hi @fukemy, sorry for the late response.

**Comment by TeruyaHaroldo** [comment]
Hi @fukemy, we released the version [1.0.6](https://github.com/Yoonit-Labs/ios-yoonit-facefy/releases/tag/1.0.6).
We just changed the platform target to `12.1`. It worked with the `Demo Project`.
Tell us if it worked to you too. 

Best regards.

**Comment by fukemy** [comment]
thanks, let me try.
